api/src/types.rs

```diff
@@ -239,6 +239,21 @@ impl OutputSwitch {
 		}
 	}
 }
+
+// Printable representation of a block
+#[derive(Debug, Serialize, Deserialize, Clone)]
+pub struct TxKernelPrintable {
+	pub excess: String,
```

Comment:
I'd add the fee, the sig, `lock_height` (if present) and features (at this time, only feature is whether it's a coinbase kernel or not).